packages/grafana-ui/src/components/uPlot/plugins/TooltipPlugin.tsx

```diff
@@ -285,3 +300,27 @@ export function positionTooltip(u: uPlot, bbox: DOMRect) {
 
   return { x, y };
 }
+
+/**
+ * Given a sort order and a value, return a function that can be used to sort values
+ * Null values are always sorted to the end regardless of the sort order
+ */
+function sortValues(sort: TooltipSortOrder) {
+  return (a: number | null | undefined, b: number | null | undefined) => {
```

Comment:
```suggestion
  return (a: any, b: any) => {
```

Reply:
@ryantxu this fn is now moved to grafana-data/arrayUtils and now has the support for string values (using `localeCompare`)